server/game/effects.js

```diff
@@ -408,6 +408,16 @@ const Effects = {
             }
         };
     },
+    cannotGainChallengeBonus: function() {
+        return {
+            apply: function(player) {
+                player.gainsChallengeBonus = false;
```

Comment:
Since 'cannot' is absolute, it's better for the flag to be named w/ the word cannot, set to `true` by the effect and be defaulted to `false` on the card (or otherwise just never set). When it isn't set up that way, it allows the potential for another effect to set `gainsChallengeBonus` back to `true`, thus negating the 'cannot' which the rules do not allow. See the other 'cannot' effects.